external/osquery_main.cpp.template

```diff
@@ -0,0 +1,32 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <osquery/sdk.h>
```

Comment:
Maybe use the `/tools/codegen` folder?

Reply:
Done! I moved it in the codegen folder and used a more meaningful name